### PR TITLE
Feat reptile page styling

### PR DESCRIPTION
**Pull request by mvic89**
Added images.
A sad attempt of making the animal content responsive.
Moved my sidebar in the current existing one.
Added a link to the click button in JS.

### CODE REVIEW

**Review by maysunktong — APPROVED** [review]
Look ok to me!

**Review by Bor-ghol — APPROVED** [review]
Looks good to me as well!

File: reptiles.html [hunk 1]
```diff
@@ -24,6 +24,10 @@
             <a href="birds.html">Birds</a>
             <a href="aboutus.html">About us</a>
         </div>
+        <div class="header-searchbar">
```
Comment (bosilf) [inline reptiles.html]: remove this, unless you're gonna add a function

File: reptiles.html [hunk 2]
```diff
@@ -24,6 +24,10 @@
...
+            <button>Login</button>
+                <input placeholder="Search...">
+        </div>
```
Comment (bosilf) [inline reptiles.html]: remove this, unless you're gonna add a function

File: reptiles.html [hunk 3]
```diff
@@ -42,20 +46,72 @@
...
                     ><i class="bx bx-home"></i> Home</a
                   >
                 </li>
-                <li><a href="reptiles.html">Reptiles</a></li>
-                <li><a href="mammals.html">Mammals</a></li>
-                <li><a href="birds.html">Birds</a></li>
-                <li><a href="aboutus.html">About Us</a></li>-->
+                <li class="tab frill-necked-lizard">Frill-necked-lizard</li>
+                <li class="tab hawksbill-turtle">Hawksbill Turtle</li>
+                <li class="tab perentie">Perentie</li> -->
```
Comment (bosilf) [inline reptiles.html]: clean up, if you're not going to use

File: reptiles.html [hunk 4]
```diff
@@ -42,20 +46,72 @@
           </label>
           <div class="sidebar">
             <nav class="sidebar-menu">
-              <ul>
-                <!--<li>
```
Comment (bosilf) [inline reptiles.html]: Your sidebar goes over the header when you scroll down. Others have put the header below the main content.
Alex:
Screenshot 2024-12-09 at 12 33 13
yours:
Screenshot 2024-12-09 at 12 33 29

**Review by bosilf — APPROVED** [review]
It's a responsive website for different screen sizes, (sidebar could be fixed but it's not breaking the page)
Title should be changed from document to reptiles.
I approve.

**Pull request merged**